src/Raven.Server/Dashboard/ThreadsInfo.cs

```diff
@@ -14,7 +14,7 @@ public class ThreadsInfo : AbstractDashboardNotification
 
         public double CpuUsage { get; set; }
 
-        public long ActiveCores { get; set; }
+        public double ActiveCores { get; set; }
```

Comment:
The problem is when running in k8s, when you can assign 0.5 cores

Reply:
This still needs to be changed. The translation from partial units to integer number of cores should be done in the script, not by RavenDB

Reply:
In a situation where the number of cores is a fraction, the process cpu calculation will be wrong. 
To calculate the process cpu usage we compare the license limit vs the actual number of cores and use the min.

Reply:
That is not our responsibility. That is what the script is for.
It would be doing that translation

Reply:
To give more context, this is important because "core" isn't well defined.
A Raspberry Pi has 4 cores, or example, but my laptop can probably beat all of them together with just one core.

The script defines what a core means in this context, not the system. 
We do ensure that we aren't going over the number of licensed core, but that is it.

Reply:
We calculate machine usage and process usage separately. If you want to delegate all that responsibility to the script, it will have to give us RavenDB's process usage as well. So the script will have to monitor the RavenDB process. It makes it much harder than just calling the K8s API metrics-server.

Reply:
I pretty sure that we have to. 
And we can do that in the same way we do now, just in the script